src/grid/index.js

```diff
@@ -164,15 +179,25 @@ class Grid {
       return false;
     }).map(size => getCleanSize(size));
 
-    for (i = 0, len = sizeList.length; i < len; i++) {
+    len = sizeList.length;
+    if(tracks === "none"){
+      len = this.getProps("maxTracks");
+      console.log(len);
+      sizeList = 'auto,'.repeat(len).split(",");
+      sizeList.pop();
+    }
+
+    for (i = 0; i < len; i++) {
       startLineNames = (nameList[i] && nameList[i].replace(/\[|\]/g, '').split(' ').filter(name => name.length).map(name => name.trim())) || [i + 1 + ''];
       endLineNames = (nameList[i + 1] && nameList[i + 1].replace(/\[|\]/g, '').split(' ').filter(name => name.length).map(name => name.trim())) || [i + 2 + ''];
 
+      //getMaxRowColumn();
       sanitizedTracks.push({
         start: i + 1,
         end: i + 2,
-        size: sizeList[i],
+        size: sizeList[i]
       });
+      //console.log(sanitizedTracks);
```

Comment:
@siddharthapaul001 If this line is not required any more, please remove this from the final PR.